ntoskrnl/io/pnpmgr/plugplay.c

```diff
@@ -218,15 +218,15 @@ IopInitializeDevice(
 
     /* Leave, if the device already exists */
     DeviceObject = IopGetDeviceObjectFromDeviceInstance(&DeviceInstance);
-    if (DeviceInstance.Buffer != NULL)
+    
+    if (DeviceObject != NULL)
     {
-        DPRINT1("Device %wZ already exists!\n", &DeviceInstance);
+        DPRINT1("Device %wZ already exists!\n", DeviceInstance);
```

Comment:
```suggestion
        DPRINT1("Device %wZ already exists!\n", &DeviceInstance);
```
This should fix a display bug too.